src/Microsoft.AspNet.Mvc.Core/ActionResults/PartialViewResult.cs

```diff
@@ -80,7 +81,13 @@ public class PartialViewResult : ActionResult
 
             using (view as IDisposable)
             {
-                await ViewExecutor.ExecuteAsync(view, context, ViewData, TempData, ContentType);
+                await ViewExecutor.ExecuteAsync(
+                    view,
+                    context,
+                    ViewData,
+                    TempData,
+                    new HtmlHelperOptions(),
+                    ContentType);
```

Comment:
`MvcOptions` isn't and shouldn't be a member of `ActionContext`.

The established pattern for accessing something off of options in an action result is to use `GetRequiredService`